src/Hl7.Fhir.ElementModel.Tests/ScopedNodeTests.cs

```diff
@@ -257,5 +271,59 @@ public async T.Task<Resource> ResolveByCanonicalUriAsync(string uri)
             public T.Task<Resource> ResolveByUriAsync(string uri) => throw new NotImplementedException();
         }
 
+        private class ElementNodeAdapter : ITypedElement, IResourceTypeSupplier
```

Comment:
Was een wrapper rond een normale `ITypedElement` die `null` teruggeeft bij `Definition` niet goed genoeg?

Reply:
Translation: wouldn't a wrapper around a normal `ITypedElement` returning `null` for `Definition` be sufficient?